src/org/exist/util/ArgumentUtil.java

```diff
@@ -0,0 +1,83 @@
+package org.exist.util;
```

Comment:
Thanks, but I am not sure they are interested. eXist args are kinda a mess, as we don't use `commands` which is kinda what is expected these days and certainly what the Jargo design would typically expect.